docs/web_reference.rst

```diff
@@ -2726,7 +2726,7 @@ application on specific TCP or Unix socket, e.g.::
 
    :param runner: a runner to serve.
 
-   :param sock: :class:`socket.socket` to listen.
+   :param sock: :func:`socket.socket` to listen.
```

Comment:
No, this param is not a function, it's a result that this function returns.
There's no label for the class specifically so let's just link "socket objects" section from the docs:
```suggestion
   :param sock: A :ref:`socket instance <socket-objects>` to listen to.
```